### PR TITLE
bugfix: Syntax problems that prevents Docker image to build

### PR DESCRIPTION
**Pull request by undercontr**
I applied a fix to the Dockerfile as the reading environment variable wasn't correct.

And explicitly executed the entrypoint.sh file with sh

### CODE REVIEW

**Comment by jimisola** [comment]
Thank you for taking the time to provide this PR. I have a busy period at the moment and it might take a little time before I can look at it.

Are you using the [pre-built image provided this project](https://github.com/unoconv/unoserver-docker/pkgs/container/unoserver-docker) or trying to built by yourself?

If you are trying to build by yourself, on what platform? Have you tried pre-built image, does it work for you?

> I applied a fix to the Dockerfile as the reading environment variable wasn't correct.

I didn't have any problems when building to my knowledge, so what error do you see/problem are you experiencing.

> And explicitly executed the entrypoint.sh file with sh

Ok. Why? Was than an error otherwise?

**Comment by undercontr** [comment]
Hello,

>Are you using the [pre-built image provided this project](https://github.com/unoconv/unoserver-docker/pkgs/container/unoserver-docker) or trying to built by yourself?

I am building the image by myself not using the pre-built images. (with a Docker installation in Windows 11)

>If you are trying to build by yourself, on what platform? Have you tried pre-built image, does it work for you?

I am sure pre-built images would work just fine but I'm not going to use the image. Just wanted to improve it since it is a great idea to use single instance for all office operations with UNO.

>I didn't have any problems when building to my knowledge, so what error do you see/problem are you experiencing.
See: [output from GitHub Action](https://github.com/unoconv/unoserver-docker/actions/runs/4769084599/jobs/8479108222#step:6:442)

>There is a [warning](https://github.com/unoconv/unoserver-docker/actions/runs/4769084599/jobs/8479108222#step:6:482) however, is that causing you problems?

>Ok. Why? Was than an error otherwise?

I had 2 errors, one was the one you warned me about ([warning](https://github.com/unoconv/unoserver-docker/actions/runs/4769084599/jobs/8479108222#step:6:482)) and the other is "file not found" error for entrypoint.sh file on container creation.

Please let me know if the answer is not sufficient.

File: Dockerfile [hunk 1]
```diff
@@ -83,4 +82,4 @@ ENV HOME="/home/worker"
 
 VOLUME ["/data"]
 
-ENTRYPOINT ["/config/entrypoint.sh"]
```
Comment (jimisola) [inline Dockerfile:86]: What is the reason that you are entering sh explicitly?

**Comment by jimisola** [comment]
> I am building the image by myself not using the pre-built images. (with a Docker installation in Windows 11)


Ok.
 

> I am sure pre-built images would work just fine but I'm not going to use the image. Just wanted to improve it since it is a great idea to use single instance for all office operations with UNO.

I'm a bit reluctant to accept the changes since I can't verify that they do work on Windows.
That said, if they work in our pipeline and official image that might be good enough - no harm done.

Also, with the next release of alpine we won't need this in the docker file as UNO will be installed correctly.

Is this a critical fix for you or would you be ok waiting for the next version of alpine and then it'll be fixed "by itself"?

> I had 2 errors, one was the one you warned me about ([warning](https://github.com/unoconv/unoserver-docker/actions/runs/4769084599/jobs/8479108222#step:6:482)) and the other is "file not found" error for entrypoint.sh file on container creation.

Ok. What fixed the file not found issue? sh explicilty?

**Pull request closed without merging**